src/coordination/include/coordination/coordinator_instance.hpp

```diff
@@ -74,6 +74,14 @@ class CoordinatorInstance {
 
   auto GetLeaderCoordinatorData() const -> std::optional<CoordinatorToCoordinatorConfig>;
 
+  auto DemoteInstanceToReplica(std::string_view instance_name) -> DemoteInstanceCoordinatorStatus;
+
+  auto TryForceResetClusterState() -> ForceResetClusterStateStatus;
```

Comment:
Where is this method used? Can we have `ForceResetClusterState` and its private method

Reply:
This is user action for force reset

Reply:
Write that down